acceptance_tests/features/steps/notify_api.py

```diff
@@ -48,3 +48,11 @@ def check_notify_api_call(notify_api_call, template, case, emitted_uac, request_
 
     test_helper.assertDictEqual(actual_personalisation, expected_personalisation,
                                 'Actual personalisation in notify API call must match expected')
+
+
+@step('we retrieve the UAC and QID from the SMS fulfilment to use for launching in RH')
```

Comment:
In this spike I used export files and just added uacs_from_actual_export_file to the context when it's checking the file.  Lighter touch to achieve the same end?

Not a massive issue though

Reply:
I did look at using an export file to get the UACs, but on preferred an SMS fulfilment because it should be quicker, it can be mostly driven by synchronous API calls where the export file has more stages of scheduling/event driven workings behind the scenes before a file reaches its destination which inevitably gives it a bit more overhead.